clients/src/main/java/org/apache/kafka/clients/consumer/internals/LegacyKafkaConsumer.java

```diff
@@ -0,0 +1,2623 @@
+/*
```

Comment:
For my understanding, this is basically a copy of the old KafkaConsumer without any changes, right?

Reply:
Yes. The only changes are the removal of the extra constructors.